pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -259,15 +266,18 @@ func (r *ObjectBucketClaimReconciler) handleDeleteClaim(key client.ObjectKey) er
 		log.Error(nil, "got nil objectBucket, assuming deletion complete")
 		return nil
 	}
+	// see if obc delete is for a newly provisioned bkt vs an existing bkt
+	isNewBkt := r.internalClient.isNewBucketByOB(ob)
```

Comment:
we're using `isNewBkt` and `isDynamicallyProvisioned` in naming to mean the same thing.  I lean toward `isDynamicallyProvisioned`